option optimize_for = LITE_RUNTIME;

message VisitOneKeyGetRewardRequest
{
 repeated uint32 route_id = 1;
}
